cmd/auth/token.go

```diff
@@ -21,9 +22,27 @@ func newTokenCommand(persistentAuth *auth.PersistentAuth) *cobra.Command {
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		ctx := cmd.Context()
+
+		var profileName string
```

Comment:
The confusing point here is when there's a mismatch between profile host and host in argument. We either should not allow to provide both at the same time or error out if there's host mismatch

Reply:
Should we throw an error if the url is provided on top of the profile?

Reply:
I'd say we throw an error when host from profile and host from args do not match. @pietern wdyt?

Reply:
I don't see the value of doing that over simply erroring because both a profile name and a host are specified.

Are there valid cases where you specify both?

If not we should make these mutually exclusive.

Reply:
We only use the profile to select the host. So currently there is no use case for passing both. I added an error if they are both specified.